src/index.html

```diff
@@ -5,8 +5,391 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+  
+        <nav class="navigation">
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link is-active">Apple</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Samsung</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Smartphones</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a
+                href="#home"
+                class="navigation__link"
+                data-qa="nav-hover"
+              >Laptops & computers</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Gadgets</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Tablets</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Photo</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="all-products">
+        <div class="card" data-qa="card">
+          <div class="card__image">
+            <img
+              src="images/imac.jpeg"
+              alt="image of display apple"
+              class="card__photo"
+            >
+          </div>
+    
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+    
+          <h2 class="card__product-code">
+            Product code: 195434
+          </h2>
+    
+          <div class="card__rating rating">
+            <div class="stars">
```

Comment:
Reuse previous code from stars task
```suggestion
            <div class="stars stars--4">
```